apps/sms/style/sms.css

```diff
@@ -81,6 +81,14 @@ form[role="dialog"][data-type="edit"] > menu {
   display: inline;
 }
 
+.bottom-tab[role="region"] {
```

Comment:
Are you sure we will need this? Because we already apply edit form visibility in message-edit-form and children should inherit it.